assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -423,27 +468,27 @@ subroutine validate_inflate_options(inf_flavor, inf_damping, inf_initial_from_re
 ! RTPS needs a single parameter from namelist: inf_initial(2).  
 ! Do not read in any files.  Also, no damping.  but warn the user if they try to set different
 ! values in the namelist.
-if (inf_flavor(2) == 4) then
-   if (inf_initial_from_restart(2) .or. inf_sd_initial_from_restart(2)) &
+if (inf_flavor(POSTERIOR) == RELAXATION_TO_PRIOR_SPREAD) then
+   if (inf_initial_from_restart(POSTERIOR) .or. inf_sd_initial_from_restart(POSTERIOR)) &
       call error_handler(E_MSG, 'validate_inflate_options:', &
          'RTPS inflation (type 4) overrides posterior inflation restart file with value in namelist', &
          text2='posterior inflation standard deviation value not used in RTPS')
-   inf_initial_from_restart(2) = .false.    ! Get parameter from namelist inf_initial(2), not from file
-   inf_sd_initial_from_restart(2) = .false. ! inf_sd not used in this algorithm
+   inf_initial_from_restart(POSTERIOR) = .false.    ! Get parameter from namelist inf_initial(2), not from file
+   inf_sd_initial_from_restart(POSTERIOR) = .false. ! inf_sd not used in this algorithm
 
-   if (.not. inf_deterministic(2)) &
+   if (.not. inf_deterministic(POSTERIOR)) &
       call error_handler(E_MSG, 'validate_inflate_options:', &
                         'RTPS inflation (type 4) overrides posterior inf_deterministic with .true.')
-   inf_deterministic(2) = .true.  ! this algorithm is deterministic
+   inf_deterministic(POSTERIOR) = .true.  ! this algorithm is deterministic
 
-   if (inf_damping(2) /= 1.0_r8) &
+   if (inf_damping(POSTERIOR) /= 1.0_r8) &
       call error_handler(E_MSG, 'validate_inflate_options:', &
                         'RTPS inflation (type 4) disables posterior inf_damping')
-   inf_damping(2) = 1.0_r8  ! no damping
+   inf_damping(POSTERIOR) = 1.0_r8  ! no damping
 endif
 
 ! enhanced inflation checks - this is before we set the subflavor in the structure.
-if (inf_flavor(1) == 5 .or. inf_flavor(2) == 5) then
+if (inf_flavor(PRIOR) == ENHANCED_SS_INFLATION .or. inf_flavor(POSTERIOR) == ENHANCED_SS_INFLATION) then
    ! check inf_sd_max_change() for valid range
    do i=1, 2
```

Comment:
A suggestion: to be consistent with the notation, the do-loop could be re-written
> do i = PRIOR, POSTERIOR

Reply:
Done.